.circleci/config.yml

```diff
@@ -509,4 +506,15 @@ executors:
     environment:
       JVM_OPTS: -Xmx3200m
       BUILDTYPE: Debug
-      GRADLE_OPTS: -Xmx4096m -Dorg.gradle.daemon=false -Dkotlin.compiler.execution.strategy=in-process
\ No newline at end of file
+      GRADLE_OPTS: -Xmx4096m -Dorg.gradle.daemon=false -Dkotlin.compiler.execution.strategy=in-process
+
+  instrumentation-test-runner:
+    parameters:
+      machine:
+        type: string
```

Comment:
should the type be ubuntu?

Reply:
no
the main idea is to use another docker image for running instrumentation tests